lib/person.rb

```diff
@@ -1,7 +1,7 @@
 require_relative '../nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rents
```

Comment:
- [ ] Again, `rental` (in your case _rents_) should be marked with `attr_accessor`, as we intend to both _access (get)_ as well as _modify (set)_ it. Kindly ensure to improve upon this aspect.